templates/invoice_entry/worklogs.html.twig

```diff
@@ -73,7 +73,7 @@
                         </td>
                         <td class="table-td">{{ worklog.worker }}</td>
                         <td class="table-td">{{ worklog.timeSpentSeconds / 60 / 60 }}</td>
-                        <td class="table-td">{{ worklog.started|date('Y-m-d') }}</td>
+                        <td class="table-td">{{ worklog.started|date('d.m.Y') }}</td>
```

Comment:
Consider setting the format globally: https://symfony.com/doc/current/reference/configuration/twig.html#format